Resources/Prototypes/_RMC14/Entities/Objects/Tools/tools.yml

```
@@ -95,6 +95,37 @@
     slots:
     - Suitstorage
 
+- type: entity
+  parent: trayScanner
+  id: CMTRayScanner
+  name: t-ray scanner
+  description: A high-tech scanning device that uses terahertz radiation to detect subfloor infrastructure.
+  suffix: trayScanner
+  components:
+  - type: Sprite
+    sprite: _RMC14/Objects/Tools/t-ray.rsi
+    layers:
+    - state: tray-off
+      map: [ "base" ]
+  - type: TrayScanner
```

This is supposed to be range 1 I think, for a 3x3
```suggestion
  - type: TrayScanner
    range: 1
```